test/support/repo.ex

```diff
@@ -6,9 +6,14 @@ defmodule Authsense.Test.Repo do
   end
   def get(_model, _id), do: nil
 
-  def get_by(_model, email: "nobody@gmail.com"), do: nil
-  def get_by(_model, email: "rico@gmail.com") do
-    valid_resource "rico@gmail.com", "foobar"
+  def get_by(model, email: email) when is_atom(model), do: Enum.find(all(), &(&1.email == email))
```

Comment:
nice one. these are stubs btw so we avoid any DB calls in tests (as a result, our tests don't need postgres or anything). you may need to make your own stubs.